hudi-flink/src/main/java/org/apache/hudi/operator/InstantGenerateOperator.java

```diff
@@ -206,7 +206,7 @@ private void doCheck() throws InterruptedException {
   private void initTable() throws IOException {
     if (!fs.exists(new Path(cfg.targetBasePath))) {
       HoodieTableMetaClient.initTableType(new Configuration(serializableHadoopConf.get()), cfg.targetBasePath,
-          HoodieTableType.valueOf(cfg.tableType), cfg.targetTableName, "archived", cfg.payloadClassName, 1);
+          HoodieTableType.valueOf(cfg.tableType), cfg.targetTableName, "archived", cfg.payloadClassName, 1, null);
```

Comment:
@wangxianghu hello,  can we get the indextype config in flink InstantGenerateOperator.java 
HoodieIndexUtils.getIndexType(hoodieWriteConfig)